code/modules/projectiles/guns/ballistic.dm

```diff
@@ -47,6 +47,8 @@
 
 	///Whether the gun will spawn loaded with a magazine
 	var/spawnwithmagazine = TRUE
+	///If the weapon fires flames out of it's back
+	var/backblast = FALSE
```

Comment:
change this to a variable that is either null or the type of the backblast bullet and set it in .proc/chamber_round according to the bullet loaded, maybe add a var to bullets for this as well.

Reply:
Did this along with all the other changes. Gives me the error that the var doesn't have a definition tied to it. Might have to do something with the other changes i made.